charts/airflow-operator/templates/_helpers.yaml

```diff
@@ -0,0 +1,149 @@
+{{ define "webhooks.clientConfig" -}}
+{{ printf "\n" }}
+{{- if .Values.webhooks.useCustomTlsCerts -}}
+caBundle: "{{ .Values.webhooks.caBundle | b64enc }}"
+{{ end -}}
+service:
+  name: {{ .Release.Name }}-airflow-operator-webhook-service
+  namespace: '{{ .Release.Namespace }}'
+  path: '{{ .path }}'
+{{- end }}
+
+# Declaring volumes for the Operator manager deployment, either use Certmanager-provided
+{{ define "manager.volumes" -}}
+{{ printf "\n" -}}
+{{- if and .Values.airgapped .Values.runtimeVersions.configMapName -}}
+- name: versions
+  configMap:
+    name: {{ .Values.runtimeVersions.configMapName }}
+{{- end }}
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) -}}
+- name: cert
+  secret:
+    defaultMode: 420
+    secretName: webhook-server-cert
+{{- else -}}
+- name: custom-certs
+  secret:
+    defaultMode: 420
+    secretName: webhooks-tls-certs
+{{- end -}}
+{{- end }}
+
+{{ define "manager.volumeMounts" -}}
+{{ printf "\n" -}}
+{{- if and .Values.airgapped .Values.runtimeVersions.configMapName -}}
+- mountPath: /etc/config/astro-runtime/metadata/versions.json
+  name: versions
+  subPath: versions.json
+{{- end }}
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) -}}
+- mountPath: /tmp/k8s-webhook-server/serving-certs
+  name: cert
+  readOnly: true
+{{- else -}}
+- mountPath: /tmp/custom-certs
+  name: custom-certs
+  readOnly: true
+{{- end -}}
+{{- end }}
+
+{{ define "manager.args" -}}
+{{ printf "\n" -}}
+- --health-probe-bind-address=:8081
```

Comment:
is this port hardcoded we have managerPort as values is this different from that vars